staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -167,6 +167,7 @@ var baseOptsWithoutStrictCost = []VersionedOptions{
 		EnvOptions: []cel.EnvOption{
 			library.AuthzSelectors(),
 		},
+		// RecognizeKeywordAsFieldName support has been introduced in 1.31 in https://github.com/kubernetes/kubernetes/blob/v1.31.0/staging/src/k8s.io/apiserver/pkg/cel/environment/environment.go#L270-L272
```

Comment:
what is the purpose of this comment?

Reply:
@cici37 is this comment still necessary?  Wondering if you could add some context here, thanks!

Reply:
Hi, please keep the comment. The comment is added for the previously added cel env setting in 1.31 in stored version and will be available in new expression starting 1.32. Since RecognizeKeywordAsFieldName support is added differently than other cel library, this comment is to keep the log clear

Reply:
I have kept this comment in as suggested